src/Token/Builder.php

```diff
@@ -169,4 +170,29 @@ public function getToken(Signer $signer, Key $key): Plain
             new Signature($signature, $encodedSignature)
         );
     }
+
+    private function formatClaims(array $claims): array
+    {
+        if (isset($claims[RegisteredClaims::AUDIENCE]) && count($claims[RegisteredClaims::AUDIENCE]) === 1) {
```

Comment:
Also, if you assume that it always is a packed array, simply use an `isset()` on index `0` and another one on index `1`